### PR TITLE
added scylla theme to docs

### PR DESCRIPTION
**Pull request by lauranovich**
**Description of your changes:**
Add Scylla Documentation Theme so we can publish with Scylla Docs look and feel. 

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**
- [x] Documentation has been updated, if necessary.
- [ ] Image has been built (`make docker-build`) on the last commit.

### CODE REVIEW

**Comment by lauranovich** [comment]
Screenshot from 2020-10-19 14-14-15

**Comment by lauranovich** [comment]
I have a weird error with the toctree - asked @dgarcia360 to look at it

**Comment by lauranovich** [comment]
@zimnx - build errror found, and fixed - ready for review!

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+# Custom sidebar templates, maps document names to template names.
+#
+html_sidebars = {'**': ['side-nav.html']}
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'ScyllaDocumentationdoc'
+
+# URL which points to the root of the HTML documentation. 
+html_baseurl = 'https://scylladb.github.io/sphinx-scylladb-theme'
```
Comment (dgarcia360) [inline docs/source/conf.py:133]: Replace for:

` html_baseurl = 'https://scylladb.github.io/scylla-operator'`

**Review by zimnx — APPROVED** [review]
LGTM

**Pull request merged**